config/tips.txt

```diff
@@ -18,33 +18,30 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
 Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
-Crates can be booby-trapped by adding wire, shocking anyone who tries to open them.
```

Comment:
@Cheridan @iamgoofball it was removed in #15262
Either accidentally or stealth removed.